python/tvm/script/builder/tir/op.py

```diff
@@ -109,6 +109,10 @@ def float64(expr):
     return _ffi_api.PrimType("float64", expr)
 
 
+def handle(name=""):
+    return _ffi_api.Handle(name)
```

Comment:
let's remove this parameter and encourage users to use `T.arg` to name variables

```suggestion
def handle():
    return _ffi_api.Handle()
```